src/pres/Settings.js

```diff
@@ -1,69 +1,143 @@
-import React from 'react'
-import {Button} from 'reactstrap'
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import { faRetweet, faTrashAlt, faFastBackward, faDesktop } from '@fortawesome/free-solid-svg-icons'
-import {Container, Row, Col} from 'reactstrap'
-import {trackEvent} from '../app/Analytics'
-import * as Constants from '../app/Constants'
+import React from 'react';
+import { Button, Col, Container, Row } from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faDesktop, faFastBackward, faMoon, faRetweet, faTrashAlt } from '@fortawesome/free-solid-svg-icons';
+import Cookies from 'js-cookie';
+
+import { trackEvent } from '../app/Analytics';
+import * as Constants from '../app/Constants';
 
 export default class SettingsView extends React.Component {
     constructor(props){
         super(props)
         window.addEventListener("keydown",this.keyHandler.bind(this))
-  
     }
+
     keyHandler(e){
         if(e.srcElement && e.srcElement.tagName === 'INPUT') {
-            return 
+            return
         }
         switch(e.keyCode) {
-          case 70: // F key
-          this.toggle('orientation')()
-          break
-          default:
+            case 70: // F key
+            this.toggle('orientation')()
             break
-
+            default:
+                break
         }
-      }
+    }
+
     toggle(eventName){
         return (()=> {
             let newValue = this.props.settings[eventName] === Constants.PLAYER_COLOR_WHITE ? Constants.PLAYER_COLOR_BLACK:Constants.PLAYER_COLOR_WHITE
             this.props.onChange(eventName, newValue)
             trackEvent(Constants.EVENT_CATEGORY_CONTROLS, "ChangeOrientation")
         })
     }
+
     analyse() {
         window.open(`https://www.lichess.org/analysis/standard/${this.props.fen}`, '_blank');
         trackEvent(Constants.EVENT_CATEGORY_CONTROLS, "AnalyzeGame")
-
     }
+
     resetAction() {
         this.props.reset()
         trackEvent(Constants.EVENT_CATEGORY_CONTROLS, "Reset")
     }
+
     clearAction() {
         this.props.clear()
         trackEvent(Constants.EVENT_CATEGORY_CONTROLS, "Clear")
     }
 
+    manualToggleDarkMode () {
+        // Cookie value should be 'true' or 'false', but if not then it will be overwritten with a valid value anyway.
+        const darkModeCookie = Cookies.get(Constants.DARK_MODE_COOKIE) === 'true';
```

Comment:
can you move this to be a part of the settings cookie that already exists. see `CookieManager.js`

Reply:
I've done this now 👍